uaclient/entitlements/tests/conftest.py

```diff
@@ -3,51 +3,68 @@
 from uaclient import config
 
 try:
-    from typing import Any, Dict, List  # noqa
+    from typing import Any, Dict, List, Optional  # noqa
 except ImportError:
     # typing isn't available on trusty, so ignore its absence
     pass
 
 
-def machine_token(entitlement_type: str) -> "Dict[str, Any]":
+def machine_token(
+    type: str,
+    *,
+    affordances: "Dict[str, Any]" = None,
+    directives: "Dict[str, Any]" = None,
+    entitled: "Optional[bool]" = True,
```

Comment:
As I mention in more detail below (I missed this definition on my first read through), this isn't `Optional`.

Reply:
dropped to bool